charts/harvester-csi-driver/100.0.2+up0.1.12/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Harvester CSI Driver
+  catalog.cattle.io/kube-version: '>= 1.18'
```

Comment:
Please update this upstream to the format `'>= 1.18.0-0 < 1.XX.0-0'` where XX is the max Kubernetes version the chart support +1. More on the reasoning here: https://github.com/rancher/charts/pull/1925